src/libs/utils.js

```diff
@@ -259,7 +259,11 @@ export const withBusyState = _.curry((setBusy, fn) => async (...args) => {
   }
 })
 
-// Performs the given effect, but only on component mount
+/**
```

Comment:
Drive-by documentation tweaking. This is my understanding based on discussion with Isaac. Please offer corrections/suggestions if you have them.